src/SFA.DAS.Data.DatabaseTests/StepBindings/InsertStepBindings.cs

```diff
@@ -0,0 +1,52 @@
+﻿using BoDi;
+using TechTalk.SpecFlow;
+using TechTalk.SpecFlow.Assist;
+using System.Collections.Generic;
+using SFA.DAS.Data.DatabaseTests.TestHelpers;
+using System;
+
+namespace SFA.DAS.Data.DatabaseTests.StepBindings
+{
+    [Binding]
+    public class InsertStepBindings
+    {
+        private InsertHelper dbHelper;
+
+        public InsertStepBindings()
+        {
+            dbHelper = new InsertHelper(ScenarioContext.Current.Get<string>("connectionstring"));
+        }
+
+        private void Insertinto(IEnumerable<dynamic> values, Action<dynamic> action)
+        {
+            foreach (dynamic value in values)
+            {
+                action(value);
+            }
+        }
+
+        [Given(@"I have DAS_LevyDeclarations")]
```

Comment:
There are already tests which make sure the underlying tables are populated when we receive events so when I discussed with Arvin we agreed that it made sense to cover the views (which have 0 tests currently) initially and if we still don't think the overall coverage is sufficient we can work them back so they cover the events as well.